### PR TITLE
Lukman10a/shop grid card

### PR DESCRIPTION
**Pull request by Lukman10a**
This card is designed to indicate its name and price(s), this was done by passing the name and price as props. Below is a screenshot;

Screenshot (11)

### CODE REVIEW

**Comment by AOMuiz** [comment]
same issue.
Also the other price the red one should be struck out as in the design

File: src/components/cards/ShopGridCard.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+const ShopGridCard = ({ url, price, name, price2 }) => {
+  return (
+    <MainContainer>
+      <ImageContainer>
+        <Image src={url} alt="" />
+      </ImageContainer>
+      <TextContainer>
+        <Name>{name}</Name>
+        <PriceContainer>
```
Comment (AOMuiz) [inline src/components/cards/ShopGridCard.js:13]: add some padding to the bottom of this price container

File: src/components/cards/ShopGridCard.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          <Price>{price}</Price>
+          <Price2>{price2}</Price2>
```
Comment (AOMuiz) [inline src/components/cards/ShopGridCard.js:15]: make this a crossed-out price as specified in the design

**Comment by Lukman10a** [comment]
done

**Comment by AOMuiz** [comment]
pull request was made to main

**Pull request closed without merging**